_example/router/router.go

```diff
@@ -10,12 +10,24 @@ func Initialize(r *gin.Engine) {
 	api := r.Group("api")
 	{
 
+		api.GET("/profiles", controllers.GetProfiles)
+		api.GET("/profiles/:id", controllers.GetProfile)
+		api.POST("/profiles", controllers.CreateProfile)
+		api.PUT("/profiles/:id", controllers.UpdateProfile)
+		api.DELETE("/profiles/:id", controllers.DeleteProfile)
+
```

Comment:
Now the order is random, because all models are fetched in parallel.

Random order causes extra diff of routers even if there is no diff of models. So models are sorted in ascii order beforehand.